packages/preview/upm-report/0.1.0/src/upm-report.typ

```diff
@@ -0,0 +1,484 @@
+#let upm-report(
+  // Cover page info
+  title: "Title of the Work",
+  author: "Author's name",
+  supervisor: "Supervisor's Name",
+  date: datetime.today(),
+  // Acknowledgements and Abstract
+  acknowledgements: none,
+  abstract-en: none,
+  keywords-en: none,
+  // License info
+  license-name: "Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International",
+  license-logo: "assets/cc-by-nc-sa.svg",
+  license-link: "https://creativecommons.org/licenses/by-nc-sa/4.0/",
+  // School configuration
+  university: "Universidad Politécnica de Madrid",
+  school-name: "E.T.S. de Ingeniería de Sistemas Informáticos",
+  school-address: "Campus Sur UPM, Carretera de Valencia (A-3), km. 7\n28031, Madrid, España",
+  school-abbr: "ETSISI",
+  report-type: "Bachelor's thesis",
+  degree-name: "Grado en Ingeniería de Tecnologías de la Sociedad de la Información",
+  school-color: rgb(32, 130, 192),
+  school-logo: "assets/etsisi-logo.svg",
+  school-watermark: "assets/upm-watermark.png",
+  // Bibliography configuration
+  bibliography-file: none,
+  bibliography-style: "ieee",
+  body,
+) = {
+  // Cover page
+  page(
+    margin: 0cm,
+    numbering: none,
+  )[
+    // Left colored bar
+    #place(
+      left + top,
+      rect(
+        width: 2cm,
+        height: 100%,
+        fill: school-color,
+      ),
+    )
+
+    // School watermark
+    #if school-watermark != none {
+      place(
+        right + bottom,
+        image(school-watermark, width: 90%),
```

Comment:
This won't work as you would expect. You should take an `image` as parameter directly, see [this guide](https://github.com/typst/packages/blob/main/docs/resources.md#paths-are-package-scoped) for more details.